### PR TITLE
#11 add cardteam component with styles

### LINKED ISSUE #11: Create about team card component
Specify card component for about team section:
1. Developer name, title (header)
2. Completed work (main content)
3. Github link (footer)

### PR DESCRIPTION
**Pull request by YaGrig**
# Description

add card component for team section, add team const for team section

Fixes #11 

## Type of change

Please choose suitable options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Screenshot is attached or not needed

### CODE REVIEW

File: src/components/CardTeam/Card.module.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    height: 100%;
+    flex-direction: column;
+    justify-content:start;
+}
+.header{
+    display: flex;
+    justify-content: space-between;
+    
+    .photo{
+        height: 112px;
```
Comment (OnlyR10) [inline src/components/CardTeam/Card.module.scss]: Please change px to rem

File: src/components/CardTeam/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classes from './Card.module.scss';
+import CardTeamInterface from './CardTeamInterface';
+
+const CardTeam: React.FC<CardTeamInterface> = (props: CardTeamInterface) => 
+    <>
```
Comment (khvalkoilya) [inline src/components/CardTeam/Card.tsx]: Please use linter

**Comment by OnlyR10** [comment]
It is necessary to pull all the current changes from the develop branch and fix conflicts

File: src/components/CardTeam/Card.module.scss (hunk 1 above)
Comment (YaGrig) [reply src/components/CardTeam/Card.module.scss]: Done

**Review by OnlyR10 — APPROVED** [review]

**Pull request merged**